mindjunkies/forums/views.py

```
@@ -17,7 +17,8 @@
 from .models import Course, ForumTopic ,LikedPost
 from .forms import ForumTopicForm, ForumCommentForm,ForumReplyForm
 
-
+from django.http import HttpResponse
```

Ruff (0.8.2)

`django.http.HttpResponse` imported but unused

Remove unused import: `django.http.HttpResponse`

(F401)